QuickViewer/src/mainwindow.ui

```diff
@@ -790,7 +790,7 @@
   </action>
   <action name="actionMaximizeOrNormal">
    <property name="text">
-    <string extracomment="Function to switch windows to maximize or normal display">&amp;Maximize or Normal</string>
+    <string extracomment="Function to switch windows to maximize or normal display">&amp;Maximize or normal</string>
```

Comment:
I have no idea what is this nor where to find it in UI...

Reply:
You can find the function, Enter the Enter Key :)